src/iwant_bot/start.py

```diff
@@ -182,26 +185,39 @@ def verify_request_token(body: dict) -> None:
 
 loop = asyncio.get_event_loop()
 
-# Created channel iwant_group10 - id: G65FE8M6K.
-# (1..9 were created and archived, but still cannot be recreate and I cannot delete them.)
-# So, we should not create to many channels?
+# from iwant_bot.slack_communicator import SlackCommunicator
 
-# test = SlackCommunicator(SUPER_TOKEN, 'U51RKKATS', 'Create channel')
-# loop.run_until_complete(asyncio.gather(test.create_private_channel('iwant_group11'),
-#                                        test.invite_people_in_private_channel())
-#                         )
+# sent_message_to_each can send message even to channels and users
+# test1 = SlackCommunicator(BOT_TOKEN, 'U52FUHD98', 'Initial message.')
+# loop.run_until_complete(test1.send_message_to_each())
 
+# sent message to multiparty group of 2 to 7 people (+ 1 iwant-bot).
+# test2 = SlackCommunicator(BOT_TOKEN, ['U51RKKATS', 'U52FUHD98', 'U52FU3ZTL'], 'Sorry spam :).')
+# loop.run_until_complete(test2.send_message_to_multiparty())
 
-# sent_message_to_each can send message even to channels and users
-test1 = SlackCommunicator(BOT_TOKEN, 'U52FUHD98', 'Initial message.')
-loop.run_until_complete(test1.send_message_to_each())
+storage = iwant_bot.pipeline.choose_correct_store()
 
 
-# sent message to multiparty group of 2 to 7 people (+ 1 iwant-bot). Need BOT_TOKEN.
-# So, this is preferable variant...
+async def loop_checker(store, delay):
+    """This is periodically called loop checker, which sends notification about unfilled requests
+    (requests close to the deadline ~ 30 seconds)."""
+    await asyncio.sleep(delay)
+    print('INFO: Loop checker...')
+    await iwant_bot.notifier.notify_not_matched_request(store, _check_storage_interval, BOT_TOKEN)
+    await loop_checker(store, _check_storage_interval)
+
+loop.create_task(loop_checker(storage, delay=_check_storage_interval))
+
+
+@worker.task()
+def storage_checker():
+    """This should be same as function loop_checker, but delegated to Celery workers."""
+    # I have some problems with async def & await.
+
+    print('INFO: Celery checker...')
+    # store = iwant_bot.pipeline.choose_correct_store()
+    # await iwant_bot.notifier.notify_not_matched_request(store)
```

Comment:
You don't need to await that, Celery runs all tasks asynchronously. 
Btw. Celery worker does print the info log from tasks with return value.